make/autoconf/flags-cflags.m4

```diff
@@ -317,6 +324,7 @@ AC_DEFUN([FLAGS_SETUP_OPTIMIZATION],
     C_O_FLAG_DEBUG_JVM=""
     C_O_FLAG_NONE="-Od"
     C_O_FLAG_SIZE="-O1"
+    C_O_FLAG_LTO="-GL"
```

Comment:
There is sometimes warning spam during linking if \-Gy is not set alongside \-GL last I remember. This probably won't break LTO builds, but it could possibly be annoying. Whether you want to add the flag is up to you, however.